### DISCUSSION TITLE
BTX-style variant

### DISCUSSION THREAD

**Discussion by erkinalp** [TheGuyDanish/CM4_MATX#66] [Ideas]
Second generations of BTX and miniBTX cases have a flipped IO layout, hence those cannot use microATX boards.
A microATX sized variant with BTX style IO layout could be useful for those.

**Comment by volkertb** [comment]
As innovative and superior it may have been to ATX, wasn't BTX a commercial failure, which hardly gained any industry traction?

**Reply by erkinalp** [reply]
A few Dell and HP office cases have that exact IO layout, a normal ATX layout will not work in those cases due to slot panel mismatch.

**Reply by volkertb** [reply]
But isn't that too small a niche to cater too in the short term, especially when there is already a shortage of knowledgeable people working on this project?

**Reply by erkinalp** [reply]
There are millions of such cases and motherboards from these cases started to fail one by one. A cheap passively cooled motherboard would be a perfect replacement board.
